### ISSUE TITLE
[11] voip_sip_webrtc error after upgrade

### ISSUE THREAD

**Issue by Vassy** [SythilTech/Odoo#28]
Hi Steven, after update today from https://github.com/SythilTech/Odoo, module voip_sip_webrtc gives this error:

```
Traceback (most recent call last):
  File "/opt/odoo11/odoo/odoo/http.py", line 650, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/opt/odoo11/odoo/odoo/http.py", line 310, in _handle_exception
    raise pycompat.reraise(type(exception), exception, sys.exc_info()[2])
  File "/opt/odoo11/odoo/odoo/tools/pycompat.py", line 87, in reraise
    raise value
  File "/opt/odoo11/odoo/odoo/http.py", line 692, in dispatch
    result = self._call_function(**self.params)
  File "/opt/odoo11/odoo/odoo/http.py", line 342, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/opt/odoo11/odoo/odoo/service/model.py", line 97, in wrapper
    return f(dbname, *args, **kwargs)
  File "/opt/odoo11/odoo/odoo/http.py", line 335, in checked_call
    result = self.endpoint(*a, **kw)
  File "/opt/odoo11/ext/modoolar/scrummer/web_syncer/http.py", line 9, in __call__
    result = self.method(*args, **kw)
  File "/opt/odoo11/odoo/odoo/http.py", line 515, in response_wrap
    response = f(*args, **kw)
  File "/opt/odoo11/ext/SythilTech/Odoo/voip_sip_webrtc/controllers/main.py", line 173, in call_kw
    value = super(DataSetInheritVoip, self).call_kw(model, method, args, kwargs, path=None)
  File "/opt/odoo11/odoo/odoo/http.py", line 515, in response_wrap
    response = f(*args, **kw)
  File "/opt/odoo11/odoo/addons/web/controllers/main.py", line 931, in call_kw
    return self._call_kw(model, method, args, kwargs)
  File "/opt/odoo11/odoo/addons/web/controllers/main.py", line 923, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/opt/odoo11/odoo/odoo/api.py", line 687, in call_kw
    return call_kw_model(method, model, args, kwargs)
  File "/opt/odoo11/odoo/odoo/api.py", line 672, in call_kw_model
    result = method(recs, *args, **kwargs)
  File "/opt/odoo11/ext/SythilTech/Odoo/voip_sip_webrtc/models/voip_server.py", line 39, in user_list
    last_activity_diff = datetime.datetime.now() - fields.Datetime.from_string(presence_user.user_id.last_web_client_activity_datetime)
TypeError: unsupported operand type(s) for -: 'datetime.datetime' and 'NoneType'

```
I tried to update voip_sip_webrtc from Odoo and from console with -u voip_sip_webrtc, but error exist. Please to your advice!

**Comment by SythilTech** [comment]
Thank you for bringing this issue to my attention, I can confirm that this issue is present in the latest version.

I have released version 1.1.3 to resolve this issue

**Comment by Vassy** [comment]
Thanks, now is ok!
